### DISCUSSION TITLE
V2 Lora Training help, V1-5 works fine

### DISCUSSION THREAD

**Discussion by mtnmecca** [bmaltais/kohya_ss#297] [Q&A]
I am successful with many trainings with V1-5.  Only issue, is that when I train on SD V2 or V2-1 with 768 base images, for training and regularization, the LoRa has no trained content.  I have tried several different training parameter options but still no success.  Any ideas of why I can't get standard V2-1 model to create a useable LoRa?  

I use all of the standard (recommended) settings when creating LoRa's and the only changes moving to V2-1 are selecting the correct "Source Model", "max resolution" to 768,768 and use 768x768 training and regularization images.  Zero success so far.  Haven't tried V2 base models yet.

**Comment by Mctwost** [comment]
Im sorry about that but im glad im not alone in this. Did you see if when you are training it it shows " text_encoder\model.safetensors not found"? while loading everything before any steps? Thats whay i get an my LoRa won't work either.

**Reply by mtnmecca** [reply]
I'll look for that "..not found" text but I believe it completes fine.  When training is complete it creates the LoRa and then I move LoRa to SD/models/Lora folder.  Within SDwebui it loads fine under "add networks" and posts no errors during txt2img with trained keyword being used.  Exact same process as V1-5 except no output resembling trained images.  I am going to try increasing the strength value higher to see if possibly way under trained.

**Reply by Yosoy-Ed** [reply]
> Im sorry about that but im glad im not alone in this. Did you see if when you are training it it shows " text_encoder\model.safetensors not found"? while loading everything before any steps? Thats whay i get an my LoRa won't work either.

I had that issue with SD 1.5, when using the droplist and selected the "runwayml/stable-diffusion-v1-5". I downloaded the .safetensor and used it instead of that, now it shows it loads ok.

My issue with SD 2.1  is that it shows "loss=nan" so I cancel the training.

**Comment by Dragoy** [comment]
I join your discussion as I also have similar problems when trying to use my Lora on a trained SD 2.1 768

**Comment by bmaltais** [comment]
I think I read that Auto1111 does not support SD2.x based LoRA. You need to use kohya's additional extension to use them.

**Reply by mtnmecca** [reply]
Well simply thank you!  I am going to retry my effort as I believe you pointed out my error.  Thanks for the information and I'll reply if/when successful.

**Reply by Dragoy** [reply]
I confirm that it works great with this extension:  https://github.com/kohya-ss/sd-webui-additional-networks

**Reply by mtnmecca** [reply]
bmaltais & dragoy.  Thanks for the information and it does indeed work by using the extension.

**Comment by Yosoy-Ed** [comment]
I have trained SD 1.5 with no problems in 512x512 and 768x768, but I am having issues with faces in full body images, so I want to train with SD 2.1, however, when I run the training it shows "loss=nan". I have changed AdamW8 to adamw and lowered the training rate as seen in other discussion without success, always "loss=nan". I am using an RTX 3070 with 8GB VRAM
